components/rhs_header_post/rhs_header_post.tsx

```diff
@@ -14,6 +14,8 @@ import FollowButton from 'components/threading/common/follow_button';
 import {browserHistory} from 'utils/browser_history';
 import Constants, {RHSStates} from 'utils/constants';
 import {t} from 'utils/i18n';
+import CRTThreadsPaneTutorialTip
+    from '../collapsed_reply_threads_tour/crt_threads_pane_tutorial_tip/crt_threads_pane_tutorial_tip';
```

Comment:
nit: can't we just use `crt` instead of `collapsed_reply_threads` to shorten import paths?